client/i18n/locales/espanol/intro.json

```diff
@@ -297,114 +297,114 @@
     }
   },
   "relational-databases": {
-    "title": "Relational Databases",
+    "title": "Bases de Datos Relacionales",
     "intro": [
-      "placeholder"
+      "marcador de posición"
     ],
     "blocks": {
       "learn-relational-databases": {
-        "title": "Learn Relational Databases",
+        "title": "Aprender Bases de Datos Relacionales",
         "intro": [
-          "placeholder"
+          "marcador de posición"
         ]
       }
     }
   },
   "apis-and-microservices": {
     "title": "APIs y Microservicios",
     "intro": [
-      "Until this point, you've only used JavaScript on the front end to add interactivity to a page, solve algorithm challenges, or build an SPA. But JavaScript can also be used on the back end, or server, to build entire web applications.",
-      "Today, one of the popular ways to build applications is through microservices, which are small, modular applications that work together to form a larger whole.",
-      "In the APIs and Microservices Certification, you'll learn how to write back end-ready with Node.js and npm (Node Package Manager). You'll also build web applications with the Express framework, and build a People Finder microservice with MongoDB and the Mongoose library."
+      "Hasta este punto, sólo has usado JavaScript en la parte de front end para añadir interactividad a una página, resolver los desafíos de algoritmos o construir un SPA. Pero JavaScript también se puede utilizar en el back end, o servidor, para construir aplicaciones web completas.",
+      "Hoy en día, una de las formas populares de construir aplicaciones es a través de microservicios, que son pequeñas aplicaciones modulares que trabajan juntas para formar un todo más grande.",
+      "En la certificación de APIs y Microservicios aprenderás cómo escribir para back- nd con Node.js y npm (Node Package Manager). También construirá aplicaciones web con el framework Express, y construirás un microservicio \"People Finder\" con MongoDB y la librería Mongoose."
```

Comment:
This looks a bit weird.  back- nd ?

Reply:
I left a message in the spanish channel, it can be fixed by a proofreader and will be corrected in GitHub next cycle

Reply:
Perfect, thanks @ieahleen. Does this make any sense as is? If it's not too confusing we can merge.

Reply:
I think the issue is just that typo, it should be fine to merge, the general sense seems correct (Spanish is not my language, but it's similar enogh)

Reply:
Cool. Good enough for me, particularly since it'll be fixed soon.